src/Microsoft.PowerShell.Commands.Utility/commands/utility/EnableDisableRunspaceDebugCommand.cs

```diff
@@ -351,7 +351,7 @@ public SwitchParameter BreakAll
         }
```

Comment:
@KirkMunro, your last commit had 2 failures in `PowerShell-CI-macos`
ActionPreference.Break tests.Other message types should break on their corresponding messages when requested.Should show 7 debugger commands were invoked
```powershell
Expected 7, but got 0.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 306
306:             $results.Count | Should -Be 7
```
ActionPreference.Break tests.ActionPreference.Break in jobs.ActionPreference.Break should break in a running job
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 352
352:             Wait-UntilTrue -sb { $job.State -eq 'AtBreakpoint' } -TimeoutInMilliseconds (10 * 1000) -IntervalInMilliseconds 100 | Should -BeTrue
```